content/en/building/condition-cards.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: "Building Condition Cards"
+title: Condition Cards
 linkTitle: Condition Cards
-weight: 12
+weight: 3
```

Comment:
```suggestion
weight: 7
```

Just to make it different from the `Tasks` section. We can review all the weights we the migration is finalized